src/MusicTheory/Notation.elm

```diff
@@ -0,0 +1,333 @@
+module MusicTheory.Notation exposing
+    ( BarAttribute(..)
+    , Bpm
+    , Chord(..)
+    , ChordNote(..)
+    , Clef(..)
+    , Notation
+    , Note(..)
+    , NoteAttribute(..)
+    , Rest(..)
+    , RestAttribute(..)
+    , Staff
+    , Voice(..)
+    , VoiceAttribute(..)
+    , VoiceElement(..)
+    , bar
+    , bass
+    , beam
+    , chord
+    , chordNote
+    , doubleLine
+    , elMap
```

Comment:
is `elMap` just an implementation detail of `voiceMap`, or will client code use it also? If not, then maybe we don't expose it?